tests/test_fluent_session.py

```diff
@@ -28,7 +28,7 @@ def print_transcript(transcript):
     print_transcript.called = False
     print_transcript.transcript = None
 
-    session.add_transcript_callback(print_transcript)
+    session._transcript.register_callback(print_transcript)
```

Comment:
@prmukherj is the `_transcript` object meant to be public - that would be my assumption. If so, shouldn't we remove the underscore?

Reply:
yes, it s public. Implemented. Thank you.